source/Calamari/Program.cs

```diff
@@ -122,7 +122,7 @@ public static void EnableAllSecurityProtocols()
             //https://central.sonatype.org/articles/2018/May/04/discontinue-support-for-tlsv11-and-below/
 
             var securityProcotolTypes =
-#if !NETCOREAPP2_0
+#if !NETCOREAPP2_2
```

Comment:
I think we should change this to a `HAS_SSL3` flag and add it here https://github.com/OctopusDeploy/Calamari/pull/402/files#diff-1ab97261a87deeb5e3c7c9f4ebf7bfcbR24

Then we don't need to keep updating it